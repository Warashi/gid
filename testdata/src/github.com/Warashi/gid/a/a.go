package a

import ( // want `not gid'ed`
	"github.com/Warashi/gid/a/local-imports"
	"errors"
	"fmt"

	_ "github.com/Warashi/wrapmsg"
)

func init() {
	fmt.Println(errors.New(local.Dummy))
}
