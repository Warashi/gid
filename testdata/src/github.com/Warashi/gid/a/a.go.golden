package a

import (
	"errors"
	"fmt"

	_ "github.com/Warashi/wrapmsg"

	"github.com/Warashi/gid/a/local-imports"
)

func init() {
	fmt.Println(errors.New(local.Dummy))
}
